### PR TITLE
Adds Dogborgs

### PR DESCRIPTION
**Pull request by Lyroy**
## About The Pull Request

Adds Dogborgs (Security K-9s, Medihounds, Janidogborgs and Borgis)
Buffs all cyborg's health by increasing it from 100 to 300 (to bring it in line with player-controlled simplemob robots).

## Why It's Good For The Game

Adds Dogborgs

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.
- [x] You added Dogborgs

## Changelog

:cl:
add: Dogborgs
/:cl:

### CODE REVIEW

**Comment by Spc-Dragonfruits** [comment]
NO!NO!NO!NO!NO!NO!NO!NO!NO!NO!NO!

**Comment by CDRDecky** [comment]
NOOOOO!!!!! NOOOOOOOOO!!!!!!!!!!!!!

**Comment by IrkallaEpsilon** [comment]
NO!

File: code/game/objects/items/devices/radio/radio.dm [hunk 1]
```diff
@@ -75,6 +77,13 @@
 		if(keyslot.independent)
 			independent = TRUE
 
+	//Skyrat edit start
```
Comment (Zonespace27) [inline code/game/objects/items/devices/radio/radio.dm:80]: L

**Comment by mazzinsanity** [comment]
How do I thumbs down

**Pull request merged**